backend/main.py

```
@@ -1593,7 +1653,36 @@ async def delete_model(model_name: str):
+            cache_dir = hf_constants.HF_HUB_CACHE
+            repo_cache_dir = Path(cache_dir) / ("models--" + hf_repo_id.replace("/", "--"))
+            
+            if not repo_cache_dir.exists():
+                raise HTTPException(status_code=404, detail=f"Model {model_name} not found in cache")
+            
+            shutil.rmtree(repo_cache_dir)
+            return {"message": f"Model {model_name} deleted successfully"}
+            
+        except HTTPException:
+            raise
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)